src/main/java/facades/SearchFacade_Impl.java

```diff
@@ -400,7 +423,7 @@ public PersonDTO_OUT addPersonWithEverything(PersonDTO_IN personDTO) {
 
             // Check if any people live in the city. 
             if (persons == null || persons.isEmpty()) {
-                throw new WebApplicationException("No Persons lives in that city.", 400);
+                throw new WebApplicationException("No Persons lives in that city.", 404);
```

Comment:
Kan du forklare den her? (Ikke at det er forkert. Jeg forstår bare ikke hvorfor :))

Reply:
Det er faktisk ikke nødvendigvis korrekt.

400 Bad Request
*The server cannot or will not process the request due to an apparent client error (e.g., malformed request syntax, size too large, invalid request message framing, or deceptive request routing).*
vs

404 Not Found
*The requested resource could not be found but may be available in the future. Subsequent requests by the client are permissible.*

Da jeg lavede det kiggede jeg kun på navnet **NOT FOUND** -- men nu bliver jeg helt i tvivl om hvilken der er korrekt.